i18n/chs/package.i18n.json

```diff
@@ -1,6 +1,7 @@
 {
   "autolispext.description": "这是适用于 AutoCAD AutoLISP 的 vscode 扩展",
   "autolispext.loadcurrentfile.title": "加载文件到 AutoCAD",
+  "autolispext.openwebhelp.title": "打开在线帮助",
```

Comment:
This is great!  I don't know how you translate these words, maybe google translator? 

In future maybe we can only change the i18n\enu\package.i18n.json file, then let the localization team in Autodesk to do left things.

Reply:
Yes, this was all google translate. I converted it to the other other language and then back. I switched to using the word 'online' instead of 'web' because it was clearly more supported across the different languages. Luckily I only needed to concern myself with 3 words at this point.

I do think language localization is a highly specialized thing. If we have the option to outsource that to pros, then that would be the best course of action.